content/99.back-matter.md

```diff
@@ -1,10 +1,14 @@
-## Acknowledgements
+## Acknowledgments
 
 We would like to thank the authors of the Deep Review who helped us test collaborative writing with Manubot.
 The authors who responded favorably to being acknowledged are Paul-Michael Agapow, Amr M. Alexandari, Brett K. Beaulieu-Jones, Anne E. Carpenter, Travers Ching, Evan M. Cofer, Dave DeCaprio, Brian T. Do, Enrico Ferrero, David J. Harris, Michael M. Hoffman, Alexandr A. Kalinin, Anshul Kundaje, Jack Lanchantin, Christopher A. Lavender, Benjamin J. Lengerich, Zhiyong Lu, Yifan Peng, Yanjun Qi, Gail L. Rosen, Avanti Shrikumar, Srinivas C. Turaga, Gregory P. Way, Laura K. Wiley, Stephen Woloszynek, Wei Xie, Jinbo Xu, and Michael Zietz.
 In addition, we thank Ogun Adebali, Evan M. Cofer, and Robert Gieseke for contributing to the Manubot template manuscript.
 We are grateful for additional Manubot discussion and testing by Alexander Dunkel, Ansel Halliburton, Achintya Rao, and other GitHub users.
 
+## Funding
+
+This work was supported by the Alfred P. Sloan Foundation in [Grant G-2018-11163](https://sloan.org/grant-detail/8501) to DSH and the Gordon & Betty Moore Foundation in [Grant GBMF4552](https://www.moore.org/grant-detail?grantId=GBMF4552) to CSG.
```

Comment:
Per https://github.com/greenelab/meta-review/pull/103#issuecomment-431373554, I find it more natural to match which authors were supported by which funding sources and ignore the grant PIs.